### DISCUSSION TITLE
New "OBO Reference Ontology Status"

### DISCUSSION THREAD

**Discussion by matentzn** [OBOFoundry/OBOFoundry.github.io#2210] [General]
As a replacement for the old "Foundry" status for ontologies the OBO Operations Committee are developing an entirely objective (automatically testable) system for checking ontologies based on the following criteria:

1. All root terms in ontology MUST be aligned with COB (moving target)
1. Ontology MUST be logically consistent with the union of their dependencies (moving target!)
1. Ontology MUST provide a base release, a variant of the ontology without external axioms (Q1 2023 to unveil exactly what that will mean)
1. Ontology MUST NOT introduce terms that belong to a “domain ontology” in scope (according to COB)
1. Some mechanism SHOULD be in place to sync with external ontologies (update deprecated class use, etc), e.g. ODK, OntoFox
1. Must pass the OBO Dashboard (not discussed with OBO Ops, but implied by criteria for library status)

The purpose of this status is to promote a set of truly interoperable, consistent, high-quality ontologies that are committed to ongoing reconciliation and collaboration to build the Grand Unified OBO Ontology, a single consistent ontology which provides unified access to biomedical concepts from all domains.

This discussion offers the opportunity for the community to say their piece.

**Comment by zhengj2007** [comment]
Can I know how the OBO Reference ontology is defined? Which OBO Foundry ontologies are considered as the OBO Reference Ontologies now? Thanks!

**Reply by matentzn** [reply]
The status does not exist at all at the moment. It will be defined by this discussion here. In my own words, whatever we call this status, it will roughly mean: "Consistent, COB-aligned component of the Grand Unified OBO Ontology".. Figuring out the exact definition is part of this discussion!

**Reply by zhengj2007** [reply]
Got it. Thanks!

**Comment by nlharris** [comment]
Note that Discussions do not come up if you search for a label (such as "attn: OFOC call"): https://github.com/OBOFoundry/OBOFoundry.github.io/labels/attn%3A%20OFOC%20call
(unless there is some way to search issues+discussions; I'm not aware of one).

Should I manually add this and the other discussion to the OFOC agenda for the next call (Jan 10)?

**Reply by matentzn** [reply]
I think if the discussion has traction, it will find its way onto the agenda. For example, since I am trying to drive this forward, it is in my interest to add it to the agenda. If I don't, it means it is not important enough, which is also good to keep the agenda small and relevant..

The discussion format is much better here so you can immediately respond to comments. Else, a linear thread in an issue becomes hard to follow.
